README.md

```diff
@@ -20,47 +20,78 @@
-1) strlen is long enough for the character array timeseries id.
-2) node would be as long as all the nodes of all the timeseries + enough special characters to separate 'holes'. 
-3) timeseries would be as long as the number of timeseries in the file.
-4) index_start_stop would be 2 or 3 depending if just the first node of a given time series or both the first and last node of a given timeseries is to be stored.
-5) time is the length of the maximum length time series in the file. 
+## Data Elements and Structure
+
+For discussion, the structure and attributes below should be what's needed to store a polygon as the `timeseries_id` of a `timeSeries` featuretype. This is using a rectangular array for the time series data and a contiguous ragged array (indexed ragged array would be silly) for the polygon nodes. In CF 2 we would expect the ragged array notation to change to a variable length data field that uses more natural 2d indexing.
+
+### Format overview with timeSeries featureType as context.
+ 
+A piece of software reading this would be expected to:  
+1) See CF-1.8 conventions  
+2) See the timeSeries featureType  
+3) Find the timeseries\_id cf\_role  
+4) Find the coordinates  
+5) See that the coordinates have a standard\_name "polygon node" to determine that these are polygons according to this new specification.  
+6) Find the variable with sample_dimension "node" to determine the count of 'samples' (nodes) along the node dimension. (See CF 1.7 Section 9.3.3, could also use format in 9.3.2 - Incomplete multidimensional array representation.)  
+7) Iterate over polygons, picking out geometries for each `timeSeries` using the count to pick out the required nodes.  
+
+Dimensionality:  
+1) strlen is long enough for the character array timeseries id.  
+2) node would be as long as all the nodes of all the timeseries + enough special characters to separate 'holes'.  
+3) polygons would be as long as the number of timeseries features in the file.  
+4) time is the length of the maximum length time series in the file.  
 
 ```
 netcdf example {
 dimensions:
-        strlen ;
-        node ;
-		timeseries ;
-		index_start_stop ;
-        time ;
+    strlen ;
+    node ;
+    polygons ;
+    time ;
 variables:
-        double polyLat(node) ;
-                polyLat:standard_name = "polygon node" ;
-        double polyLon(node) ;
-                polyLon:standard_name = "polygon node" ;
-		int polyIndex(timeseries, index_start_stop) ;
-				polyIndex:cf_role = "node_index" ;
-        char timeseriesID(timeseries, strlen) ;
-                timeseriesID:cf_role = "timeseries_id" ;
-        int data(timeseries, time) ;
-                data:coordinates = "time polyLat polyLon" ;
+    int crs() ;
+    double polyLat(node) ;
+				polyLat:standard_name = "polygon y node" ;
+				polyLat:grid_mapping = "crs"
+    double polyLon(node) ;
+				polyLon:standard_name = "polygon x node" ;
+				polyLon:grid_mapping = "crs"
+    int polyIndex(polygons) ;
```

Comment:
Since we switched from indexed to contiguous ragged array, polyIndex should be renamed to polyNodeCount. Also, I'm a little uncomfortable using the name "node" since it's really "nodes plus some indicator of holes" though I don't have a better name.